modules/govuk_elasticsearch/templates/es-prune-snapshots.erb

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+# Redirect stdout and stderr to syslog
+exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
+
+ES_REPOS=(<%= @repositories %>)
+LIMIT=<%= @es_snapshot_limit %>
+
+for ((i=0; i < ${#ES_REPOS[@]}; i++)); do
```

Comment:
You can probably simplify the loop with `for repo in ${ES_REPOS[@]}; do` and then use $repo later.